packages/connectivity/lib/connectivity.dart

```diff
@@ -15,6 +15,7 @@ import 'package:meta/meta.dart';
 /// None: Device not connected to any network
 enum ConnectivityResult { wifi, mobile, none }
 
+/// Discover network connectivity configurations: Distinguish between WI-FI and cellular, check WI-FI status and more.
```

Comment:
I would comment the `@visibleForTesting` things too, just to be safe. It's fine for the comment just to say something like "This is just exposed for testing purposes and should not be altered by users of the plugin."

Reply:
Going to land with this change.